### PR TITLE
Linters.

### PR DESCRIPTION
**Pull request by carlylechia**
On this branch, I
       installed and ran linters for both HTML and CSS;
       created a .html file and added the text, "Hello Microverse!";
       styled the HTML file with CSS;
       ignored some files using .gitignore.

### CODE REVIEW

File: .github/workflows/.stylelintrc.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "extends": ["stylelint-config-standard"],
+    "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
+    "rules": {
+      "at-rule-no-unknown": null,
+      "scss/at-rule-no-unknown": true,
+      "csstree/validator": true
+    },
+    "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css"]
```
Comment (micheaol) [inline .github/workflows/.stylelintrc.json]: Please ensure the linter is set up correctly for this repository. Please check how to correctly set-up linter [here](https://github.com/microverseinc/linters-config)

File: .github/workflows/index.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Hello Microverse!</title>
+</head>
+<body>
+    <h1 class="header">
+        Hello Microverse!
+    </h1>
+</body>
+</html>
```
Comment (micheaol) [inline .github/workflows/index.html]: Please ensure that the index.html file is created as a separate file in your root directory. Please remove it from the .github folder.

File: .github/workflows/styles.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.header {
+    color: aqua;
+}
```
Comment (micheaol) [inline .github/workflows/styles.css]: Please ensure that the style.css file is created as a separate file in your root directory. Please remove it from the .github folder.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @carlylechia , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️

- _Please check the comments under the review._
- Please ensure your pull request has a descriptive title telling you what changes are inside. Please check details [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)
- _Remember that your branch name should be descriptive (e.g.: refactor-authentication, user-content-cache-key, make-retina-avatars), so that others can see what is being worked on. Kindly change the branch name from 'others' to a more meaningful name. Please watch this [video ](https://www.youtube.com/watch?v=QV0kVNvkMxc)for more understanding_


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by elmar8287** [comment]
Hi @carlylechia , 

Well done on making the suggested changes.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**